src/main/java/io/cdap/plugin/MongoDBConfig.java

```diff
@@ -124,6 +131,10 @@ public String getPassword() {
     return password;
   }
 
+  public boolean isUseSRV() {
```

Comment:
adding 'is' as a prefix doesnt make much sense, keep it just connectUsingSRVString